tools/vccexe/vccdiscover.nim

```diff
@@ -0,0 +1,98 @@
+## VCC compiler backend discovery Utility
+##
+## Module to discover the path to the vcvarsall utility of a VCC compiler backend.
+## The module supports discovery for either the latest recognizable version (default)
+## or discovery of a specific VCC compiler backend version.
+##
+## This module can also produce a standalone command-line executable.
```

Comment:
@Araq what exactly do you mean? The VCCEXE does actually call the vccdiscover module. That's what happens with the `--vccversion` command-line option. As it states in [tools/vccexe/vccexe.nim#L30](https://github.com/couven92/Nim/blob/devel/tools/vccexe/vccexe.nim#L30), the argument value for `--vccversion` is passed directly on to `vccdiscover`

I could however, merge the vccexe and vccdiscover modules together, if that is what you had in mind...